### ISSUE TITLE
Clean text up for `User who uploaded relationship` label for managed files 

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#6549]
This is a follow-up to https://github.com/backdrop/backdrop-issues/issues/6440

The current help text is as follows:
`The user account that uploaded the file. If more fields than UID are needed, add the "File: User who uploaded" relationship`

I think the remaining issue is not actually for the file managed UID field anymore, but instead for the name of the relationship, whic is `File: User who uploaded`. But once we change the name of that relationship, we'll also need to update this help text to match. 

From @indigoxela 
> Yeah, a follow-up re wording / naming might make sense. Follow-up, because naming issues usually need a little more discussion. ("Author who uploaded" wouldn't make sense... 😉)

From @quicksketch 
> I'm not sure if it's an official code style thing, but I don't like talking "directly to the user" using "you" or "your" unless talking about their user account. i.e. I would avoid "You have saved your view." but "You can update your password." is fine.

From @klonos 
> however, do note that when changing strings that use the word user on its own, we usually try to change that to either user account or person, whichever makes more sense and is appropriate.

**Comment by klonos** [comment]
`File: Account used to upload the file` sounds accurate, although not sure if too wordy.

**Comment by bugfolder** [comment]
How about `File: uploader account`?

**Comment by NormPlum** [comment]
I made a PR for this. The Redirect module just uses the name "User ID", so I used that name here too.

<img width="741" height="112" alt="Image" src="https://github.com/user-attachments/assets/da27926f-5658-43b9-b739-a54f2c4ebafc" />
